### ISSUE TITLE
load colors before loading "after"?

### ISSUE THREAD

**Issue by slmjkdbtl** [macvim-dev/macvim#722]
I noticed that colors are not defined before the `after` folders getting sourced, have to use autocmds to change the colors on load

**Comment by eirnym** [comment]
Does it reproducable on base Vim (from https://github.com/vim/vim)?

**Comment by slmjkdbtl** [comment]
No, with other color schemes I'm specifying `colorscheme *` in my rc file, but macvim uses a builtin colorscheme and it seems to be sourced after the `after` folders loaded

**Comment by eirnym** [comment]
in file `/Applications/MacVim.app/Contents/Resources/vim/gvimrc` MacVim indeed loads it's colorscheme:

```vim
" Load the MacVim color scheme.  This can be disabled by loading another color
" scheme with the :colorscheme command, or by adding the line
"   let macvim_skip_colorscheme=1
" to ~/.vimrc.
if !exists("macvim_skip_colorscheme") && !exists("colors_name")
    colorscheme macvim
endif
```

I explicetly load my own colorscheme in my `vimrc`.
